### PR TITLE
[11.0][UPD] travis file

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fix travis file.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
-python:
-  - "3.5"
 
 env:
-  - VERSION="11.0" LINT_CHECK="1"
-  - VERSION="11.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="11.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  global:
+  - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - PHANTOMJS_VERSION="latest"
```
Comment (pedrobaeza) [inline .travis.yml]: This line is not needed for not adding extra build time.

Comment (MiquelRForgeFlow) [reply .travis.yml]: ok

**Comment by MiquelRForgeFlow** [comment]
Ready :green_apple:

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking this

**Pull request merged**